docs/pages/guides/ci-cd-pipelines.mdx

```diff
@@ -162,4 +164,370 @@ RUN chmod +x /bin/loft
 
 ### Login with Access Keys
 
-<LoginAccessKeysFragment/>
\ No newline at end of file
+<LoginAccessKeysFragment/>
+
+## Using GitHub Actions
```

Comment:
\## Integrations
\### GitHub Actions

=> Turn current h3 headlines into h4 headlines on the entire page.